test/Makefile

```diff
@@ -209,3 +213,11 @@ $(RESULTS_DIR)/d_do_test$(EXE): d_do_test.d $(RESULTS_DIR)/.created
 	$(QUIET)$(DMD) -conf= $(MODEL_FLAG) -unittest -run d_do_test.d -unittest
 	$(QUIET)$(DMD) -conf= $(MODEL_FLAG) -od$(RESULTS_DIR) -of$(RESULTS_DIR)$(DSEP)d_do_test$(EXE) d_do_test.d
 
+start_separate_tests: $(RESULTS_DIR)/.created $(RESULTS_DIR)/d_do_test$(EXE)
```

Comment:
I suppose using the `COMPILE_SEPARATELY` flag doesn't do what you want?

Reply:
Didn't know about it. I'm going to revert all my Makefile changes and use it instead.

Reply:
It's documented at the top of this makefile.